packages/ai/core/tool/mcp/mcp-client.ts

```diff
@@ -76,13 +78,16 @@ class MCPClient {
   > = new Map();
   private serverCapabilities: ServerCapabilities = {};
   private isClosed = true;
+  private strictMode = false;
```

Comment:
The value is set in the constructor. Would prefer `private strictMode: boolean` to prevent the impression that the `false` setting here matters.